themes/unl/templates/RecordDriver/SolrDefault/core.phtml

```diff
@@ -260,15 +293,52 @@
         <th><?=$this->transEsc('Online Access')?>: </th>
         <td>
           <? foreach ($urls as $current): ?>
-            <a href="<?=$this->escapeHtmlAttr($this->proxyUrl($current['url']))?>"><?=$this->escapeHtml($current['desc'])?></a><br/>
+            <a href="<?=$this->escapeHtmlAttr($this->proxyUrl($current['url']))?>"><?=$this->escapeHtml(($current['url']==$current['desc']?$this->proxyUrl($current['desc']):$current['desc']))?></a><br/>
           <? endforeach; ?>
           <? if ($openUrlActive): ?>
             <?=$openUrl->renderTemplate()?><br/>
           <? endif; ?>
         </td>
       </tr>
       <? endif; ?>
-
+      
```

Comment:
AltTitles and Eresource Notes have shown up here, so I think we'll be ok with that part